### DISCUSSION TITLE
Continuous integration

### DISCUSSION THREAD

**Discussion by MattF-NSIDC** [nsidc/tech-radar#12] [Techniques]
We made a decision two weeks ago that GitHub actions should be our go-to for new CI work. If a project is already using CircleCI, there's no need to migrate.

**Comment by jwfuller** [comment]
In the past, we have used

- Bitbucket Pipelines
- Jenkins for CI
- CircleCI

**Comment by rmarow** [comment]
Note: GitHub actions are free for public repositories only.

**Comment by trey-stafford** [comment]
We like to use GitHub Actions for new repositories and we migrate from CircleCI for existing projects if time allows.

Jenkins is still used as needed to perform CI actions on NSIDC internal infrastructure.
